.gitignore

```diff
@@ -0,0 +1,6 @@
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+*.Rproj
```

Comment:
I was convinced at the unconf to leave it the *.Rproj.....there are project level options that can be set in there, such as tab widths, text encodings. 

So, if you use the RStudio package builder...devtools...or this new one "mason"....they all include the *.Rproj in their git commits for that reason (you do need to remember to include it in the .Rbuildignore). That being said, our group has not committed these files in the past and we've survived OK....so I am by no means passionate about the issue. But, if we want to follow the other group's convention, then I'd take that out.